tests/plugins/copyformatting/manual/edgefocus.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: bug, 4.9.0, copyformatting, 1458
+@bender-ui: collapsed
+@bender-ckeditor-plugins: copyformatting,toolbar,wysiwygarea,floatingspace,elementspath,image2,widget,justify,div
```

Comment:
You could easily run this test with only `copyformatting,toolbar,wysiwygarea,image2`. Noe need for additional plugins.